frontend/components/Badge/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Image from '@shopgate/pwa-common/components/Image';
+import {
+  PRODUCT_DETAILS_PAGE_LOCATION,
+  PRODUCT_CARD_LOCATION,
+  PRODUCT_GRID_LOCATION,
+} from '../../constants';
+import styles from './styles';
+
+/**
+ * @param {Array} badgeInfo Array of badge image urls
+ * @returns {JSX}
+ */
+const CardBadge = ({ location, badgeInfo }) => {
+  const imageClass = classNames(
+    { [styles.cardImage]: location === PRODUCT_CARD_LOCATION },
+    { [styles.gridImage]: location === PRODUCT_GRID_LOCATION },
+    { [styles.pdpImage]: location === PRODUCT_DETAILS_PAGE_LOCATION }
+  );
+
+  const images = badgeInfo.map((image, index) => (
+    <Image className={imageClass} src={image} alt="" key={index.toString()} />
```

Comment:
I was wrong about using the library Image component. It adds a background color to the image. It is probably not what is wanted. 

![image](https://user-images.githubusercontent.com/11764719/66229175-dd1aee00-e6a6-11e9-901d-e5d0b234951e.png)